docs/tutorials/gpu_setup_tutorial.rst

```diff
@@ -0,0 +1,228 @@
+will assume that ``device=cuda:0``. *Note: ‘cuda:0’ is technically only
+required if you have more than one GPU, if ``device='cuda'`` then
+PyTorch will use the default cuda device.*
+
+Now that we have our device set, lets create some data-filled tensor
+objects from the mock ALMA dataset. This is a multi-channel dataset
+which is represented as a data cube. Here we will use the central
+channel of the cube for demonstration purposes, this corresponds to
+``central_chan=4``.
+
+.. code:: ipython3
+
+    from astropy.utils.data import download_file
+    import numpy as np
+    from mpol import gridding, coordinates
+    fname = download_file(
+        'https://zenodo.org/record/4498439/files/logo_cube.npz',
+        cache=True,
+        )
+    d = np.load(fname)
+    coords = coordinates.GridCoords(cell_size=0.03, npix=180)
+    central_chan = 4
+    gridder = gridding.Gridder(
+        coords=coords,
+        uu=d['uu'][central_chan],
+        vv=d['vv'][central_chan],
+        weight=d['weight'][central_chan],
+        data_re=d['data_re'][central_chan],
+        data_im=d['data_im'][central_chan],
+    )
+    dataset = gridder.to_pytorch_dataset()
+
+Now lets create a SimpleNet Neural Network that we will train with our
+data. A SimpleNet, as defined by MPoL, is a combination of the most
+common modules for imaging. For a visual and for more detailed
+information, see the `Optimization
+Loop <https://mpol-dev.github.io/MPoL/tutorials/optimization.html>`__
+tutorial or the MPoL SimpleNet `Source
+Code <https://mpol-dev.github.io/MPoL/_modules/mpol/precomposed.html#SimpleNet>`__.
+MPoL’s SimpleNet class is part of the ``mpol.precomposed`` library.
+
+.. code:: ipython3
+
+    from mpol.precomposed import SimpleNet
+    model = SimpleNet(coords=coords, nchan=dataset.nchan)
+
+We are now ready to move our model and data to the GPU. This process is
+rather simple, PyTorch tensor objects are given a ``tensor.to(device)``
+functionality that will move the data to the specific device. One can
+also use the ``tensor.cuda()`` to move the tensor to the default CUDA
+device. Both of these methods return a *copy* of the object on the GPU.
+In our case, ``device='cuda:0'``, so we will move the *SimpleNet* object
+(``model``) and our data (``dataset``), a GriddedDataset MPoL object, to
+the GPU. Once we move ``model`` to the GPU, then we will create an
+optimizer for the network.
+
+Also defined below is the ``config`` dictionary. It is a set of
+parameters used to scale our Neural Network’s learning. These specific
+parameters are borrowed from the `Cross Validation
+Tutorial <https://mpol-dev.github.io/MPoL/tutorials/crossvalidation.html>`__
+due to the strong cross validation score they result in. For more
+details on these variables, see the `Cross Validation
+Tutorial <https://mpol-dev.github.io/MPoL/tutorials/crossvalidation.html>`__.
+
+*Note: GriddedDataset objects also enherits a
```

Comment:
Minor text edit: "enherits" --> "inherits"

Reply:
Thank you for going over this!!!

Reply:
Happy to help!